README.md

```diff
@@ -133,6 +133,30 @@ assert_eq!(offsets, [0, 64, 192]);
 
 ```
 
+Supports writing to uninitialized memory as well.
+
+```rust
+use std::mem::MaybeUninit;
+use encase::{ShaderType, DynamicStorageBuffer};
+
+let mut byte_buffer: Vec<MaybeUninit<u8>> = Vec::new();
+
+let mut buffer = DynamicStorageBuffer::new_with_alignment(&mut byte_buffer, 64);
+let offsets = [
+    buffer.write(&[5.; 10]).unwrap(),
+    buffer.write(&vec![3u32; 20]).unwrap(),
+    buffer.write(&glam::Vec3::ONE).unwrap(),
+];
+
+// SAFETY: Vec<u8> and Vec<MaybeUninit<u8>> share the same layout.
+let byte_buffer: Vec<u8> = unsafe { std::mem::transmute(byte_buffer) };
```

Comment:
This relies on the unspecified layout of `Vec` and might result in UB if the compiler ever reorders the fields of one type but not the other. The proper way would be to use `Vec::from_raw_parts`. See the second to last example of `std::mem::transmute`'s documentation.

Reply:
I'll switch this to a boxed slice then since Vec::into_raw_parts is unstable and this is shopped to be an example.

Reply:
Done!